stable/compass/values.yaml

```diff
@@ -26,7 +26,7 @@ app:
 
   service:
     annotations:
-      {}
+      projectcontour.io/upstream-protocol.h2c: 80,http
```

Comment:
Can we avoid having this in default values? This is a setup specific annotation and can be added at the time of deployment.

Reply:
I guess that makes sense. cc: @mabdh

Reply:
updated thanks guys @rohilsurana @StewartJingga